recipes/importlab/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "importlab" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d855350d19dc10a17aabd2fe6f4b428ff1a936071f692fbf686a73694d26a51c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - networkx
+    - python
+    - six
+
+test:
+  imports:
+    - importlab
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/importlab/
```

This should refer to the project, not PyPI
```suggestion
  home: https://github.com/google/importlab
```